src/OpenTelemetry.Instrumentation.StackExchangeRedis/OpenTelemetry.Instrumentation.StackExchangeRedis.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0" />
```

Comment:
This line is not needed. See https://www.nuget.org/packages/OpenTelemetry.Api.ProviderBuilderExtensions/1.8.1#dependencies-body-tab